modules/computer/boots.c

```diff
@@ -30,11 +30,11 @@ scan_boots_real(void)
     scan_os(FALSE);
 
     if (!computer->os->boots)
-      computer->os->boots = g_strdup_printf("[%s]\n", _("Boots"));
+      computer->os->boots = g_strdup_printf("");
```

Comment:
It's OK to leave this like so; it looks weird, but it's a reminder to fully convert this to use `struct Info`.